App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Profile/Experience.jsx

```diff
@@ -1,15 +1,759 @@
-﻿/* Experience section */
-import React from 'react';
+﻿import React from 'react';
 import Cookies from 'js-cookie';
+import { Button, Grid, Label, Icon, Table, Form } from 'semantic-ui-react';
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
+import DateInput from 'semantic-ui-calendar-react';
 
 export default class Experience extends React.Component {
     constructor(props) {
         super(props);
+        
+        const exp = props.experienceData ? props.experienceData : [{ company: "", position: "", responsibilities: "", start: null, end: null, id: "", userId: "", isDeleted: "" }]
+
+
+        this.state = {     
+            start: '',
+            sdate: {},
+            showEditSection: false,
+            experienceData: exp,  
+            show: false,
+            id: 1,
+            addrow: false,                       
+            newExp: {
+                company: "",
+                position: "",
+                responsibilities: "",
+                start: null,
+                end: null,
+                id: "", userId: "", isDeleted: "" 
+            }
+
+        }
+
+        this.addDisplay = this.addDisplay.bind(this)
+        this.closeDisplay = this.closeDisplay.bind(this)
+        this.handleChange = this.handleChange.bind(this)
+        this.handleChangeDate = this.handleChangeDate.bind(this)
+        this.loadData = this.loadData.bind(this);
+        this.addRecord = this.addRecord.bind(this)
+        this.updateExperience = this.updateExperience.bind(this);
+        this.deleteExperience = this.deleteExperience.bind(this);
+        this.formatDate = this.formatDate.bind(this);  
+        this.handleDateChange = this.handleDateChange.bind(this);
+    };
+    
+    componentDidUpdate(prevProps) {
+                
+        if (this.props.experienceData !== prevProps.experienceData) {
+                    
+        }
+    }
+
+    componentDidMount() {
+        
+        this.loadData();
+
+    }
+
+    loadData() {
+        return new Promise((resolve, reject) => {
+
+            var link = 'https://mvpstandard-p.azurewebsites.net/profile/profile/getExperience'
+            
+
+            var cookies = Cookies.get('talentAuthToken');
+            $.ajax({
+                url: link,
+                headers: {
+                    'Authorization': 'Bearer ' + cookies,
+                    'Content-Type': 'application/json'
+                },
+                type: "GET",
+                contentType: "application/json",
+                dataType: "json",
+                success: function (res) {
+                    if (res.success == true) {
```

Comment:
If you are expecting a boolean response. **res.success** is enough and you don't need to use == true